CITATION.md

```diff
@@ -0,0 +1,78 @@
+# Citing and logo
+
+## Citing pandas
+
+If you use *pandas* for a scientific publication, we would appreciate citations to the published software (1) and one of the following papers:
+
+- *pandas* version 1.0.1 published on Zenodo (please find us on Zenodo and replace with the citation for the version you are using)
+```
+@software{reback2020pandas,
+  author       = {Jeff Reback and
```

Comment:
How was this list determined? I'd say perhaps keep it as "Wes McKinney and the pandas development team" or just "The pandas development team". Curious what others think though.

Reply:
yes this should be just the pandas development team

Reply:
This is how you guys released it on Zenodo. I will update it in the PR and if you wish, I think you can even go into Zenodo and adjust that so that people fetch it correctly when they get their BibTeX entry from there.